commonmark-android-test/app/build.gradle

```diff
@@ -11,7 +11,7 @@ if (testPropertiesFile.canRead()) {
 
 android {
     compileSdkVersion 15
-    buildToolsVersion "23.0.2"
+    buildToolsVersion "21.1.1"
```

Comment:
Now I realize that you don't use up-to-date version of build tools intentionally. So please ignore my previous comment about tools version.